docs/src/main/asciidoc/datasource.adoc

```diff
@@ -525,10 +519,8 @@ public class MyProducer {
 [[datasource-multiple-single-transaction]]
 === Use multiple datasources in a single transaction
 
-By default, XA support on datasources is disabled,
-and thus a transaction may include at most one datasource.
-Attempting to access multiple non-XA datasources in the same transaction
-would result in an exception similar to this:
+By default, XA support on datasources is disabled. Therefore, a transaction might include no more than one datasource.
```

Comment:
Grammar: it's "may" here, isn't it?

Feel free to ignore, I'm not a native speaker, but this feels wrong.

```suggestion
By default, XA support on datasources is disabled. Therefore, a transaction may include no more than one datasource.
```

Reply:
+1

Reply:
Not a native speaker myself.
I know it may sound funny, but the IBM style guide doesn't allow us to use the word "may" in technical documentation unless it is used to indicate permission. It clearly states that we should use "might" instead when describing a possibility.
Have a look:
![image](https://github.com/user-attachments/assets/5f15a1d5-27a2-4ba7-a181-22e272de95c4)

Reply:
Well, this *is* about a permission. It's not allowed to include more than one datasource.

Reply:
`It's not allowed to include more than one datasource.`
Why is it not stated directly in the text, then?

Reply:
> `It's not allowed to include more than one datasource.` Why is it not stated directly in the text, then?

It is, the original text says "Attempting to access multiple non-XA datasources in the same transaction
would result in an exception similar to this" - that implies that it's not allowed to enlist more datasources in the transaction.

Reply:
OK